apps/lfx-one/src/app/modules/badges/badges-dashboard/badges-dashboard.component.scss

```
@@ -0,0 +1,31 @@
+
+.badge-filter-btn {
+  @apply relative inline-flex items-center gap-1.5 px-2.5 py-1 w-fit
+    rounded-md border border-gray-300 text-xs font-medium text-gray-700
+    hover:bg-gray-50 transition-colors bg-white cursor-pointer;
+}
+
+.badge-filter-dot {
+  @apply absolute -top-1 -right-1 w-2 h-2 rounded-full bg-blue-500;
+}
+
+:host ::ng-deep [data-testid='badges-grid'] {
```

Stylelint (17.7.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)